### PR TITLE
Method to check if security is configured.

### PR DESCRIPTION
**Pull request by prasantakumar82**
This small change is required to know if srd is passed or not.

### CODE REVIEW

File: diagnostic/client/src/main/java/org/terracotta/diagnostic/client/connection/DiagnosticServiceProvider.java [hunk 1]
```diff
@@ -55,4 +55,8 @@ public DiagnosticService fetchDiagnosticService(InetSocketAddress address, Durat
       throw new DiagnosticServiceProviderException(e);
     }
   }
+
+  public boolean isSecurityConfigured() {
+    return securityRootDirectory != null && !securityRootDirectory.isEmpty();
```
Comment (saurabhagas) [inline diagnostic/client/src/main/java/org/terracotta/diagnostic/client/connection/DiagnosticServiceProvider.java:60]: To check security, it's enough to check is the security root directory is null - you don't need to do an `isEmpty()` check. With that, you wouldn't need this change.

Comment (prasantakumar82) [reply diagnostic/client/src/main/java/org/terracotta/diagnostic/client/connection/DiagnosticServiceProvider.java:60]: This method is used in enterprise code.

Comment (saurabhagas) [reply diagnostic/client/src/main/java/org/terracotta/diagnostic/client/connection/DiagnosticServiceProvider.java:60]: Like I said, it's enough to check `securityRootDirectory != null` to check if security is or isn't configured. With that, the check becomes trivial, and the method you added will not be needed.

**Comment by mathieucarbou** [comment]
not needed anymore

**Pull request closed without merging**